person.rb

```diff
@@ -10,6 +10,12 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.person = self
   end
```

Comment:
- [ ] It will be visible if we pass in a book and date as a parameter in the add_rental method so that a book at a particular date should be rented out to a person, that's a`dd_rental(book, date)` and 
the person itself should be sent as a parameter to create the new rental achieving the 'has-many' association that's `Rental.new(date, book, self)`